Makefile

```diff
@@ -25,10 +25,8 @@ help:
 	@echo "  help              to print this help message. (Default)"
 	@echo "  install           to install app by running 'pip install -e .'"
 	@echo "  develop           to install with additional development requirements."
-	@echo "  start             to start $(APP_NAME) service as daemon (background process)."
-	@echo "  stop              to stop $(APP_NAME) service."
```

Comment:
Is there a good reason to remove `stop` and `status` make targets?

Reply:
I can't they there is a good reason ... but currently I don't have the daemon mode for the pywps service implemented in the pywps command line interface. We can bring it back in later steps.

Reply:
How long is "later steps"?  If it's a while, in order to not forget, could we either add a TODO comment to the code or open an issue?

Reply:
I have opened an issue: #96.